tox.ini

```diff
@@ -4,8 +4,11 @@ skip_missing_interpreters = True
 
 [testenv]
 install_command = pip install --process-dependency-links {opts} {packages}
-deps = .[docs,test]
 commands =
+# install documentation and testing requirements
+# NOTE: Don't use 'deps = .[docs,test]' tox option since we want Tox to install
+# the package from sdist first
+    pip install --process-dependency-links .[docs,test]
```

Comment:
Currently, no. However, using `--process-dependency-links` in both places in `tox.ini` ensures that a developer can easily add a git version of a dependency without knowing all the details of Tox.